README.md

```diff
@@ -51,7 +51,8 @@ build images.  The flexibility of building images without Dockerfiles allows
 for the integration of other scripting languages into the build process.
 Buildah follows a simple fork-exec model and does not run as a daemon
 but it is based on a comprehensive API in golang, which can be vendored
-into other tools.
+into other tools.  Podman uses Buildah's golang API for the podman-build
```

Comment:
I think I'd put this new verbiage at the end of the following paragraph.  WDYT?

Reply:
I put it here to have it close to the explanation of Buildah also being/exposing an golang API. But I can move it down :+1:.

Reply:
It is also vendored into the OpenShift Image Builder.

Reply:
How about 

For example,  Podman uses Buildah's golang API for the podman-build

Reply:
just noted, missing a '.' at the end.  Rather than "podman-bulid", I think I'd go 
```
`podman build` command.
```